checklists/crossbrowser-test.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Crossbrowser Test
+category: Project Management
+date: "2020-03-11"
+tags: ['test', 'responsive']
+description: Complete the crossbrowser test to check your project according to the different type of devices and browsers.
+---
+
+- [ ] Test the project via different browsers on desktop  
+We do crossbrowser test on desktop side via different browsers such as Safari, Chrome, Firefox.  
+
+- [ ] Test the project via a bigger screen monitor  
+We control the design of project is suitable for many measure or not.  
```

Comment:
```suggestion
- [ ] Test the project in bigger screen size    
So that you can start to check the responsiveness of the website. We should also control the design of the project in different screen sizes.
```